modules/ROOT/pages/introduction/intro-config-use-studio.adoc

```diff
@@ -0,0 +1,160 @@
++
+** *Start delay*
++
+The amount of time to wait before initiating the flow for the first time after the app starts, based on the time unit specified in the *Time unit* field
++
+** *Time unit*
++
+The time unit for the values of *Frequency* and *Start Delay*
++
+* If you select *Cron*, enter the cron expression in the *Expression* field and the time zone in the *Time Zone* field. For
+ the format and valid values for the *Time Zone* field, see the Javadoc for the Java TimeZone class.
++
+. To configure metadata for the payload, variables, and attributes, select the *Metadata* tab and add the appropriate metadata.
+
+For more information about Scheduler, see xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint (Trigger)].
+
+=== Use a Connector-Specific Input Source
+
+If a connector has associated input sources, you can use a connector-specific input source to initiate the flow. Each connector input source has a unique configuration.
+
+For example, you can use Amazon SQS Receive Messages as an input source for Amazon SQS Connector. This input source initiates a flow each time the specified Amazon SQS queue receives a new message.
+
+To configure a connector-specific input source:
+
+. In the Mule Palette view, search for the connector and select the input source.
++
+For example, search for *Amazon SQS* and select the *Receive Messages* input source:
++
+.Searching for the Amazon SQS Receive Messages input source
+image::introduction/intro-config-studio-select-specific.png[Amazon SQS Receive Messages input source]
++
+. Drag the input source onto the canvas.
+. In the configuration screen, click the plus sign (*+*) next to the *Connector configuration* field to add a global element for the input source.
+. Enter values for the required fields.
+. On the *Advanced* tab next to the *Proxy* tab, specify a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[configureConnector]]
+== Configure the Connector
+
+To configure a connector in your Mule app, first configure a global element that can be used by all instances of that connector in the flow:
+
+. In the Mule Palette view, search for the connector and select the desired operation.
+. Drag the operation onto the canvas to the right of the input source. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to add a global element for connector.
+. Specify the global element properties for the connector.
+. On the *Advanced* tab in the global element properties, specify a reconnection strategy, if supported.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+After you configure a global element, configure the other fields for the connector.
+
+For information about configuring a specific connector, see the user guide and reference guide for that connector.
+
+== See Also
+
```

Comment:
This topic repeats some of the information in the Overview topic. You might take a look at the overview and be sure that it does not contain specifics that belong better in the specific configuration task topics.

Reply:
I will.